src/docs/product/performance/event-detail.mdx

```diff
@@ -56,11 +56,9 @@ The trace view may be limited to one project at a time if you're on the [Team pl
 Some spans within a transaction may be the parent of another transaction. Under these circumstances, some Span IDs will have a "View Child" or "View Children" button. These will potentially lead to another transaction or a list of transactions.
 
 <Alert>
-  <markdown>
 
-    Traversing between parent and child transactions is only available on the [Business plan](https://sentry.io/pricing/). Further, project permissions may affect access to some of these transactions.
+Traversing between parent and child transactions is only available on the [Business plan](https://sentry.io/pricing/). Further, project permissions may affect access to some of these transactions.
```

Comment:
This indent created a code block. See for new rendering https://sentry-docs-git-feat-relay-stable.sentry.dev/product/performance/event-detail/:

![image](https://user-images.githubusercontent.com/1433023/95722347-18fe7100-0c74-11eb-81d5-6678d3a3e5bf.png)

Reply:
To clarify, "cross project view" is the requirement that's only available on business plan.

Reply:
Thanks, @dashed. Since I'm just trying to fix the rendering bug, should we follow up with another PR to clarify?